app/javascript/redux/greetings/greetingsSlice.js

```diff
@@ -0,0 +1,46 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+export const fetchGreetings = createAsyncThunk(
+  'greetings/fetchGreetings',
+  async () => {
+    try {
+      const response = await fetch('/api/v1/greeting');
+      if (!response.ok) {
+        throw new Error('Network response was not ok');
+      }
+      const json = await response.json();
+      console.log(json);
```

Comment:
- [Optional] Hi @aamir-asaram, after debugging our codes, let's kindly always get rid of this line of code since it can expose some important raw data to our users.